__tests__/Unit/Components/Tasks/Card.test.tsx

```diff
@@ -114,9 +127,7 @@ describe('Task card', () => {
     test('should show edit button when ALT key is long pressed', () => {
         const { getByTestId, queryByTestId } = renderWithRouter(
             <Provider store={store()}>
-                <isUserAuthorizedContext.Provider value={true}>
-                    <Card {...DEFAULT_PROPS} />
-                </isUserAuthorizedContext.Provider>
+                <Card {...DEFAULT_PROPS} />
             </Provider>,
             {}
         );
```

Comment:
Shouldn't test for if user is not authorized be done as well?

Reply:
I will be addressing this issue in this ticket

https://github.com/Real-Dev-Squad/website-status/issues/670